backend/src/v2/compiler/argocompiler/dag.go

```diff
@@ -72,6 +72,15 @@ func (c *workflowCompiler) DAG(name string, componentSpec *pipelinespec.Componen
 		if err != nil {
 			return err
 		}
+		// Add Parallelism Limit if present
+		parallel := int64(kfpTask.GetIteratorPolicy().GetParallelismLimit())
+		if parallel > 0 {
+			currentParallelism := dag.Parallelism
+			if currentParallelism == nil || parallel > *currentParallelism {
+				dag.Parallelism = &parallel
+			}
+		}
+
```

Comment:
@gmfrasca can we get a test for this [here](https://github.com/kubeflow/pipelines/blob/master/backend/src/v2/compiler/argocompiler/argo_test.go)? 

Something that confirms that when in the pipeline dag we specify parallelism, it gets set in the argo workflow?

Reply:
added a .json and .yaml to the existing argocompiler test to validate parallelism items are set correctly in the argo workflow